test/unit/models/users.test.js

```diff
@@ -47,6 +47,22 @@ describe("users", function () {
       expect(data.roles.in_discord).to.equal(false);
     });
 
+    it("should add the user collection with archived property false and in_discord property true in roles", async function () {
```

Comment:
This test is failing, please fix it 👍🏻

Reply:
Hey @kotesh-arya, fixed this issue. Please review again.